sdk/cosmos/azure-cosmos-spark_3-0_2-12/CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing
```

Comment:
Why does module have its own ```CONTRIBUTING.md``` file. Is there anything here that isn't better covered in the root one?

Reply:
Regarding Java vs. Scala - Jonathan and I had an offline conversation about this. Using Scala for us is non-negotiable (it this is or shoudl become a no-go for you guys we will need to move the source code to a separate GitHub repository like today all Spark connectors are (EventHub, Kusto/DataExplorer, CosmosDB etc.). Writing a Spark connector with Java technically became possible with recent changes in Spark Data Source V2 - but the vast majority of Spark connectors (whether the built-in connectors for Kafka, CSV files, Sql/Jdbc etc. or external ones like Cassandra, EventHub, CosmosDB, Kusto/DataExplorer) are written in Scala - and Scala simply makes it much easier to test, customers would expect samples in Scala etc. 
In the conversation Jonathan and I had it sounded like it would be ok to give this a try - we deal with any "extras" that are necessary due to using Scala - and if this really should become an issue for the Azure central team we can always move out the source code to a separate github repository then. For us the trade-off is that we need to accept certain limitation and additional review cycles - but being able to reuse most of the engineering infrastructure for CI, release etc. is certainly a huge benefit. I personally also think other teams (Sql, EventHub, Kusto/DataExplorer) might eventual benefit from having a way to host their Spark connectors in this repository because ultimately they all do the same thing - wrap their Java SDKs in a Scala Spark connector to expose the capabilities of their Service to Spark users.

Reply:
I have removed the CONTRIBUTING.MD - makes sense to just keep the global one...